### ISSUE TITLE
About the ReID code

### ISSUE THREAD

**Issue by xieyuhaoli** [Pirazh/Vehicle_Key_Point_Orientation_Estimation#10]
Excuse me, are you sure that the lower part of your branch only concat the seven channels of the key point output?Why is my accuracy lower than the baseline in the reproduction process?Do you know why?

**Comment by Pirazh** [comment]
Yes, as stated in the paper we select 7 heat maps of size 56 by 56 corresponding to the 7 most prominent key-points given the orientation of the vehicle, mount a gaussian kernel to dilate the heat maps, concatenate them with the 56 by 56 intermediate feature maps of the resnet model, and pass them to the remaining resnet blocks.

**Comment by xieyuhaoli** [comment]
Yes, I selected seven channels for feature connection according to the output direction results corresponding to different channels.
I would like to know the details of your training. Baseline trained 20 epochs, how many times did you train plus key points?





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Pirazh/Vehicle_Key_Point_Orientation_Estimation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年3月12日(星期五) 凌晨0:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Pirazh/Vehicle_Key_Point_Orientation_Estimation] About the ReID code (#10)





 
Yes, as stated in the paper we select 7 heat maps of size 56 by 56 corresponding to the 7 most prominent key-points given the orientation of the vehicle, mount a gaussian kernel to dilate the heat maps, concatenate them with the 56 by 56 intermediate feature maps of the resnet model, and pass them to the remaining resnet blocks.

**Comment by Pirazh** [comment]
Note that the key-point and orientation estimation section of AAVER is trained separately. Once you trained the 2steps of keypoint and orientation estimation module, you freeze its weights. This module performs in inference mode while training the rest of AAVER. Next, the baseline, which is the first path of AAVER is pertained on CompCars dataset and then is fine-tuned on target datasets (either Veri or Vehicle ID) for 20 epochs. After all this, you put the baseline and keypoint estimation modules in the AAVER pipeline and train the orientation conditioned feature extraction branch.

**Comment by Pirazh** [comment]
You can find all the training details in the implementation details section.

**Comment by Pirazh** [comment]
Since there is no more discussion here, I close this issue.
